front/app/components/Form/Components/Controls/LinearScaleControl.tsx

```diff
@@ -38,6 +38,7 @@ const LinearScaleControl = ({
         flexDirection="row"
         gap="16px"
         overflow="visible"
+        flexWrap="wrap"
       >
         {uischema.options?.minimum_label && (
```

Comment:
I realized that the linear scale doesn't work well for mobile views (it also looks strange if it's too long and needs to be wrapped). I've messaged Ben and we'll see if we can fix it before the release. For now I've just added a wrap so all info displays on the screen at least.